POC/AzureVision.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    key = "d341921d724e44bda113bc343e88d476"
</pre><p>Better not to hardcode the key but rather put as environment variable and read from that, with instructions inline to tell the user to set the key environment variable. Problem is that once you merge this into the public repository, the key will leak.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-herbarium/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>